recipes-extended/onl/onl.inc

```diff
@@ -169,9 +172,10 @@ do_install() {
   mv ${D}${libdir}/libonlp-platform-defaults.so ${D}${libdir}/libonlp-platform-defaults.so.1
   ln -r -s ${D}${libdir}/libonlp-platform-defaults.so.1 ${D}${libdir}/libonlp-platform-defaults.so
 
-  # platform file
-  install -d ${D}${sysconfdir}/onl
-  echo "${ONL_PLATFORM}-r${ONIE_MACHINE_REV}" > ${D}${sysconfdir}/onl/platform
+  # install libonlp shared library (includes TODO)
```

Comment:
This is the third consecutive section titled "install libonlp shared library (includes TODO)" in this file. That's not very helpful, is it? And there is another section right of them without such a comment, where a libonlp shared library is _also_ installed.

There are too many "(includes TODO)" in this file already. It's not even obvious from the line what the TODO means. The idea appears to be that there should be a list of the static libraries included in the shared library (because such a list exists a few lines above). Nobody expanded the existing TODOs since they were added in early 2019. I don't see how this would be different here. Don't add another TODO: either provide the information that you think should be there, or omit the TODO.